pkg/operator/operator.go

```diff
@@ -209,6 +230,40 @@ func isOwned(obj interface{}) (bool, error) {
 	return ok, nil
 }
 
+func (optr *Operator) eventHandlerSingleton(f func(interface{}) bool) cache.FilteringResourceEventHandler {
+	workQueueKey := fmt.Sprintf("%s/%s", optr.namespace, optr.name)
+	addToQueue := func(obj interface{}) {
+		glog.V(4).Infof("Event: Add")
+		logResource(obj)
```

Comment:
Are we possibly being a bit noisy with this logging? Do we need the Event Add log?